activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   Add support for disabling transactions per database.
+
+    This allows you to disable transactions for a specific database, which can
+    be useful for read-only connections.
+
+    ```ruby
+    ActiveRecord::TestFixture.setup_transactions do
+      # return a hash of database names and whether transactions should be used
+      { "primary" => true, "readonly" => false }
```

Comment:
I think a simple array of the ones you don't want to use transactions for is better than supporting the hash. 

1) The hash is a less ergonomic API. By default transactions are on, so having an API that lets you turn them on when passing true for any doesn't change behavior, seems confusing, and not generally useful.
2) the hash makes the internal code way more complicated than it needs to be.

I'm thinking:

```
ActiveRecord::TestFixtures.ignore_transactions = [:primary]
```